app/controllers/application_controller.rb

```diff
@@ -6,7 +6,7 @@ class ApplicationController < ActionController::Base
 
   def mobile_device?
     if session[:mobile_override]
-      session[:mobile_override] == "1"
+      session[:mobile_override] == '1'
```

Comment:
I think Its on by default and I just copied the default config file to the
repo

On Sat, Apr 4, 2015 at 3:02 AM, nikolas <notifications@github.com> wrote:

> In app/controllers/application_controller.rb
> <https://github.com/mcfiredrill/forttree/pull/33#discussion_r27744345>:
>
> > @@ -6,7 +6,7 @@ class ApplicationController < ActionController::Base
> >
> >    def mobile_device?
> >      if session[:mobile_override]
> > -      session[:mobile_override] == "1"
> > +      session[:mobile_override] == '1'
>
> Alright, I'll configure rubocop to prefer double quotes
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mcfiredrill/forttree/pull/33/files#r27744345>.
>



-- 
-Tony Miller
github.com/mcfiredrill
@freedrull
freedrool.us